### PR TITLE
Update to details on FOSS and WIF Communities of IIT

### PR DESCRIPTION
**Pull request by Santhusha-Mallawatantri**
Updating the details of the FOSS Campus Club of IIT and the WIF Community of IIT

### CODE REVIEW

File: Women-In-Foss/IIT.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# IIT - Informatics Institute of Technology
+ 
+ A set of highly skilled and enthusiastic individuals who come together to build a community of free and open source development under the guidance of Mozilla Sri Lanka. At Mozilla Campus Club of IIT, we create awareness of all the free resources in the technology field and pave the way to many global challenges while aiming to interact with others to share, learn, teach, protect and build upon platforms, to make things that matter. Our mission is to encourage evolving developers to embrace their passion for technology.
+
+
+## Members
+
+* Santhusha Mallawatantri (Club Captain) [@LinkedIn](https://www.linkedin.com/in/santhusha-mallawatantri/)
```
Comment (dhanushkac) [inline Women-In-Foss/IIT.md]: @Santhusha-Mallawatantri are you the club lead of Women in FOSS as well?

Comment (dhanushkac) [inline Women-In-Foss/IIT.md]:
```suggestion
- **Santhusha Mallawatantri (Club Captain)**
```

Please follow the `template.md`

File: cc/IIT/2021-2022.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# IIT - Informatics Institute of Technology
+ 
+ A set of highly skilled and enthusiastic individuals who come together to build a community of free and open source development under the guidance of Mozilla Sri Lanka. At Mozilla Campus Club of IIT, we create awareness of all the free resources in the technology field and pave the way to many global challenges while aiming to interact with others to share, learn, teach, protect and build upon platforms, to make things that matter. Our mission is to encourage evolving developers to embrace their passion for technology.
+
+
+## Members
+
+* Santhusha Mallawatantri (Club Captain) [@LinkedIn](https://www.linkedin.com/in/santhusha-mallawatantri/)
```
Comment (dhanushkac) [inline cc/IIT/2021-2022.md]: The same applies here. Please follow `template.md`

**Review by dhanushkac — CHANGES_REQUESTED** [review]

File: Women-In-Foss/IIT.md (hunk 1 above)
Comment (Santhusha-Mallawatantri) [reply Women-In-Foss/IIT.md]: @dhanushkac Yes for Mozilla, FOSS and WIF I'm the lead

**Comment by Santhusha-Mallawatantri** [comment]
@dhanushkac check the requested changes made

**Review by dhanushkac — APPROVED** [review]

**Pull request merged**